command/server/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 	JSONRPCAddr       string     `json:"jsonrpc_addr"`
 	Telemetry         *Telemetry `json:"telemetry"`
 	Network           *Network   `json:"network"`
+	PubSubBuffersSize uint64     `json:"pubSub_buffers_size"`
```

Comment:
You use a mix of camelCase and snake case for the JSON, it should only be in snake_case

Reply:
Resolving because we agreed internally these shouldn't be configurable yet